packages/pie-inline-choice/docs/demo/.pie/item.entry.js

```diff
@@ -0,0 +1,80 @@
+
+/** Auto generated by /usr/local/lib/node_modules/pie/lib/apps/item/entry.js */
```

Comment:
shouldn't be in the repo - might have to ignore `.pie`?

Reply:
Done